### ISSUE TITLE
Make Practical Logistics 2 Sapphire Ore/Sapphires more easily accessible. 

### ISSUE THREAD

**Issue by TSM-EVO** [Darkosto/SkyFactory4Issues#401]
## Context

Practical Logistics 2 looks like a pretty great mod which is really under the radar. 

Right now, Sapphire Ore is only available through the Foregoing Laser. Once a player gets sapphires, they can be produced with Resource Hogs, but the Foregoing Laser is gated behind pink slime, which requires RF. 

My suggestion is to enable a player to create Sapphires more easily than with use of a Laser. Perhaps something like an alloy of diamond/lapis poured into a gem cast? Or putting a diamond or emerald into blue slime in a cauldron. 

Other similar mods in the pack include: 

* Integrated Dynamics, which is only gated behind iron and menril.
* XNet, which is only gated behind gold, and requires RF. 

The one different between those mods and Practical Logistics 2 has a wireless item access, although somewhat limited, and without crafting.

**Comment by JackRuby1** [comment]
You can get sapphire ore in the Lost Cities and the Twilight(Not 100% sure on this one), the Lost Cities dimension is suppose to be like a mining dimension.

**Comment by Jack-McKalling** [comment]
Correct, I've found Sapphire ore in Lost Cities, although very rarely. Never in Twilight Forest (yet).
